src/gui/mainwindow.cpp

```diff
@@ -877,36 +877,36 @@ void MainWindow::createKeyboardShortcuts()
     m_ui->actionOpen->setShortcut(QKeySequence::Open);
     m_ui->actionDelete->setShortcut(QKeySequence::Delete);
     m_ui->actionDelete->setShortcutContext(Qt::WidgetShortcut);  // nullify its effect: delete key event is handled by respective widgets, not here
-    m_ui->actionDownloadFromURL->setShortcut(Qt::CTRL + Qt::SHIFT + Qt::Key_O);
-    m_ui->actionExit->setShortcut(Qt::CTRL + Qt::Key_Q);
+    m_ui->actionDownloadFromURL->setShortcut(Qt::CTRL | Qt::SHIFT | Qt::Key_O);
+    m_ui->actionExit->setShortcut(Qt::CTRL | Qt::Key_Q);
 #ifdef Q_OS_MACOS
     m_ui->actionCloseWindow->setShortcut(QKeySequence::Close);
 #else
     m_ui->actionCloseWindow->setVisible(false);
 #endif
 
-    const auto *switchTransferShortcut = new QShortcut(Qt::ALT + Qt::Key_1, this);
+    const auto *switchTransferShortcut = new QShortcut(Qt::ALT | Qt::Key_1, this);
```

Comment:
Do you mind adding parentheses along the way?
`(Qt::ALT | Qt::Key_1)`

Reply:
> Do you mind adding parentheses along the way?
> `(Qt::ALT | Qt::Key_1)`

Done.